src/Hint/Bracket.hs

```diff
@@ -25,7 +25,7 @@ yes = \ x -> (x && x) -- @Suggestion \x -> x && x
 no = \(x -> y) -> z
 yes = (`foo` (bar baz)) -- @Suggestion (`foo` bar baz)
 yes = f ((x)) -- @Warning x
-main = do f; (print x) -- @Suggestion do f print x
+main = do f; (print x) -- @Suggestion print x
```

Comment:
We can't just casually delete `f` out of context. The correct fix is `do f; print x`?